content/for-developers/tutorials/inner-loop/05-expose-app/expose-app.md

```diff
@@ -1,99 +1,165 @@
+      - `path: /`: Defines the path for incoming requests. In this case, requests that match the root path ("/") will be routed to the corresponding backend service.
 
-a. Retrieve the port assigned to the service by running the following command:
+      - `servicePort: 'http'`: Specifies the port or target port of the backend service that should handle the incoming requests. The value 'http' represents the name of the port defined in the Service resource associated with the backend service.
 
-`oc get service nordmart-service`
+    > You can change or add any configuration for the ingress. To see more configurations [click](https://github.com/stakater/application.git).
 
-b. Identify the port assigned to the NodePort field. This port will be used to access your application externally.
+1. Run `tilt up` at the root of your directory.
 
-c. Obtain the IP address of any node in the cluster:
+1. Let's go to the cluster and see your ingress resource and the route associated with it in your namespace:
 
-`oc get nodes -o wide`
+    ![find ingress](images/find-ingress.png)
 
-d. Combine the node's IP address with the assigned port to access your application. For example: `http://<node-ip>:<assigned-port>`
+    We can see there is an `Ingress` resource created with our mentioned `Host`.
 
-### LoadBalancer
+1. Now let's see if `Ingress` has created the route. Go to the routes under the networking section:
 
-If your OpenShift cluster is running in a cloud environment that supports load balancers, you can use the LoadBalancer service type. This type automatically provisions a load balancer to distribute traffic to your application:
+    ![find-ingress-route](images/find-ingress-route.png)
 
-a. In your deployment configuration YAML file, define a service with the type LoadBalancer:
+   Here you can see the route created and the service that is associated with `Ingress`.
 
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: nordmart-service
-spec:
-  type: LoadBalancer
-  ports:
-    - port: 80
-      targetPort: 8080
-  selector:
-    app: nordmart
-```
+1. Copy the route with the copy icon and add `/api/review/329199` at the end of the route.
 
-In the above example, a service named `nordmart-service` is defined with the LoadBalancer type, exposing port 80 and forwarding traffic to port 8080 of the application.
+    Now you can access the application externally via ingress.
 
-b. After applying the configuration, OpenShift will provision a load balancer in the cloud environment, which will route external traffic to your application.
+    ![output ingress](images/output.png)
 
-c. Obtain the load balancer IP address or domain name provided by your cloud provider to access your application. For example: `http://<load-balancer-ip>:<port>`
+By following the tutorials above, you can successfully expose your application deployed on SAAP, using the example application "`stakater-nordmart-review-api`".
```

Comment:
changes will be deployment to saap when cicd updates the gitops with the latest chart after pr is merged